package.json

```diff
@@ -1,7 +1,7 @@
 {
   "homepage": "http://dinofrontend.github.io/ameria_ui_library",
   "name": "dino_ui_tools",
-  "version": "1.3.9",
+  "version": "1.3.12",
```

Comment:
latest version is 1.3.9, so change 12 to 10

Reply:
Already 1.3.11